tsconfig.json

```diff
@@ -0,0 +1,38 @@
+{
+    "compilerOptions": {
+        "target": "es5",
+        "noImplicitAny": true,
+        "noImplicitReturns": true,
+        "out": "built/common-sim.js",
+        "rootDir": ".",
+        "newLine": "LF",
+        "declaration": true,
+        "sourceMap": false
+    },
+    "files": [
+        "libs/accelerometer/sim/accelerometer.ts",
```

Comment:
By default all .ts files are included if no files are listed. We only want the sims files included, so we have to list them explicitly.